### PR TITLE
NEWS: Add 1.16.0 and 1.17.0-rc1 sections - v1.17.x

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Preparation for making 1.17.0 releases

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2021. ALL RIGHTS RESERVED.
+## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2024. ALL RIGHTS RESERVED.
```
Comment (rakhmets) [inline NEWS:2]:
```suggestion
## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001. ALL RIGHTS RESERVED.
```

Comment (yosefe) [reply NEWS:2]: Why do you suggest to revert it?

Comment (rakhmets) [reply NEWS:2]: It should be only the year of creation according to the guidance.

**Review by rakhmets — APPROVED** [review]

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2021. ALL RIGHTS RESERVED.
+## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001. ALL RIGHTS RESERVED.
```
Comment (tvegas1) [inline NEWS:2]: shouldn't we add -2024?

Comment (tvegas1) [reply NEWS:2]: just saw previous comments

**Review by tvegas1 — APPROVED** [review]

File: NEWS (hunk 2 above)
Comment (rakhmets) [reply NEWS:2]: It should be only the year of creation according to the guidance.

**Pull request merged**